### ISSUE TITLE
Links

### ISSUE THREAD

**Issue by drjakob** [roschaefer/story.board#428]
As a reporter
I want to use Links in my text components with easy markup
in order to link to sources.

**Comment by drjakob** [comment]
@roschaefer: long ago we had this idea. Is this feature still implemented?

**Comment by DavidLichtenberger** [comment]
In regards to safety, we'd probably have a little more control with Markdown.
As I don't know how the input of the text components happens (what options are and are not provided by the text input form etc.) I'd say it's up to you.
Either way is fine for me – from a technical perspective, neither rendering HTML straight from the JSON nor converting Markdown is a problem.

**Comment by roschaefer** [comment]
There is no option for html input (and it wasn't supposed there would be) some smart editors apparently thought that would be included - I guess, plain HTML is easier for users to debug (copy+paste from a the source code of a website)? Although markdown is a little safer from the developers perspective.

**Comment by drjakob** [comment]
So we solve links with markdown. Instagram-Images as in #458 are HTML embed Links (like https://www.instagram.com/p/bNd86MSFv6/).

**Comment by DavidLichtenberger** [comment]
I'm sorry, but I don't see any advantage in going both ways simultaneously – can't we (you) decide on one way to go (either markdown or html), so we don't have to implement both?

**Comment by drjakob** [comment]
Yes, lets decide for HTML because all embed codes (youtube, instagram) will be HTML.

**Comment by DavidLichtenberger** [comment]
Fine with me :)
